playbook/terminator.yml

```diff
@@ -19,45 +19,45 @@
         balancer_vhosts:
           - name: codesearch
             upstreams: [ 's1.codesearch.aelve.com' ]
-            listen_data: "{{ codesearch_port }} ssl"
+            listen: "{{ codesearch_port }} ssl"
             server_name: "codesearch.aelve.com"
             error_page: false
             server_without_port: true
 
           - name: codesearch-staging
             upstreams: [ 's1.staging.codesearch.aelve.com' ]
-            listen_data: "{{ codesearch_staging_port }} ssl"
+            listen: "{{ codesearch_staging_port }} ssl"
```

Comment:
In my opinion this API will be clearer:
![image](https://user-images.githubusercontent.com/5605372/55625160-04a50280-57b9-11e9-990f-57f11566dfc4.png)

Reply:
And for default port make the `listen_port` and `target_port` optional.
Something like:
![image](https://user-images.githubusercontent.com/5605372/55627208-a4b15a80-57be-11e9-95be-697b4b1815b7.png)

Reply:
`ssl on` is [obsolete](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl)